frontend/src/enum/SeverityEnum.ts

```diff
@@ -2,4 +2,15 @@ export enum Severity {
     success = 'success',
     warning = 'warn',
     error = 'error',
-}
\ No newline at end of file
+}
+
+export enum ErrorCode {
+    conflict = 'conflict',
+    selfGrantProhibited = 'selfGrantProhibited',
+    default = 'default',
+}
```

Comment:
Maybe better use Typescript convention for enum: **PascalCase** or **CONSTANT_CASE** for enum's property.